contracts/Registry.sol

```diff
@@ -186,14 +188,20 @@ contract Registry is IRegistry, ERC721Holder, ERC1155Receiver, ERC1155Holder {
             ensureIsNull(lending);
             ensureTokenNotSentinel(uint8(cd.paymentToken[i]));
             bool is721 = cd.nftStandard[i] == IRegistry.NFTStandard.E721;
+
+            uint16 _lendAmount = uint16(cd.lendAmount[i]);
+
+            if(is721) require(_lendAmount == 1, "ReNFT::lendAmount should be equal to 1");
```

Comment:
1) if we aren't using `is721` in any other place in this function, then we can just go

```solidity
if (cd.nftStandard[i] == IRegistry.NFTStandard.E721) require(_lendAmount == 1, ...)
```

after the above (if applicable), let's go: `forge fmt` or whatever it was to format (in a separate commit)

Reply:
is721 is used when emitting the event